bin/scrape_software_versions.py

```diff
@@ -4,7 +4,7 @@
 import re
 
 regexes = {
-    'nf-core/smrnaseq': ['v_nfcore_smrnaseq.txt', r"(\S+)"],
+    'nf-core/smrnaseq': ['v_pipeline.txt', r"(\S+)"],
     'Nextflow': ['v_nextflow.txt', r"(\S+)"],
```

Comment:
Will need to add in all the other software used in the pipeline (in `environment.yml`) here too.

Reply:
ok!. what about the R packages? or with the R version is enough?

Reply:
We dont generally add in the individual packages but you can add them in if you want to :smile: Bioconductor packages are normally tied to a particular version of `R` anyway.

Reply:
Looks good now 👍

Reply:
lol, I think I am good not adding individual packages :P